corehq/apps/reports/standard/deployments.py

```diff
@@ -749,8 +791,13 @@ def _pct(val, total):
         query = self.user_query().run()
 
         aggregations = query.aggregations
-        last_submission_buckets = aggregations[0].normalized_buckets
-        last_sync_buckets = aggregations[1].normalized_buckets
+        if self.selected_app_id:
+            (last_submission_buckets, last_sync_buckets) = [
+                agg.filtered_agg.date_histogram.normalized_buckets for agg in aggregations
+            ]
```

Comment:
nit: I believe using for loop here makes it harder to read. May be better to keep it simpler like the else block.

Reply:
[65b3675](https://github.com/dimagi/commcare-hq/pull/34178/commits/65b36757a36579afce6f205999c07e1f2c8dad74)